File: aliyun-python-best-practise/使用API管理RDS实例/createDBinstance_demo.py
from aliyunsdkcore.client import AcsClient
from aliyunsdkrds.request.v20140815.CreateDBInstanceRequest import CreateDBInstanceRequest
client = AcsClient('access_key_id', 'access_key_secret', 'region_id')
request = CreateDBInstanceRequest()
request.set_PayType("Postpaid")
request.set_SecurityIPList("10.23.12.27/25")
request.set_DBInstanceNetType("Internet")
request.set_DBInstanceStorage("20")
request.set_DBInstanceClass("rds.mysql.t1.small")
request.set_EngineVersion("5.6")
request.set_Engine("MySQL")
response = client.do_action_with_exception(request)
print(response)

File: aliyun-java-best-practise/src/main/java/com/aliyun/bestpractise/CreateDBInstanceDemo.java
package com.aliyun.bestpractise;

import com.aliyuncs.DefaultAcsClient;
import com.aliyuncs.IAcsClient;
import com.aliyuncs.exceptions.ClientException;
import com.aliyuncs.exceptions.ServerException;
import com.aliyuncs.profile.DefaultProfile;
import com.aliyuncs.rds.model.v20140815.*;

/**
 * 使用API创建RDS实例
 */
public class CreateDBInstanceDemo {

    public static void main(String[] args) {
        DefaultProfile profile = DefaultProfile.getProfile("<regionId>", "<accessKeyId>", "<accessSecret>");
        IAcsClient client = new DefaultAcsClient(profile);

        CreateDBInstanceRequest request = new CreateDBInstanceRequest();
        request.setPayType("<payType>");
        request.setSecurityIPList("<securityIPList>");
        request.setDBInstanceNetType("<dBInstanceNetType>");
        // your dBInstanceStorage
        request.setDBInstanceStorage(0);
        request.setDBInstanceClass("<dBInstanceClass>");
        request.setEngineVersion("<engineVersion>");
        request.setEngine("<engine>");

        try {
            CreateDBInstanceResponse response = client.getAcsResponse(request);
        } catch (ServerException e) {
            e.printStackTrace();
        } catch (ClientException e) {
            System.out.println("ErrCode:" + e.getErrCode());
            System.out.println("ErrMsg:" + e.getErrMsg());
            System.out.println("RequestId:" + e.getRequestId());
        }

    }
}
